svir/test/unit/test_load_oq_engine_output_as_layer.py

```diff
@@ -271,20 +271,31 @@ def test_load_dmg_by_asset_aggregate_by_zone(self):
             zonal_layer_plus_stats_first_feat.attributes(),
             expected_zonal_layer_first_feat.attributes())
 
-    def _test_export(self, expected_file_name):
+    def _test_export(self):
         _, exported_file_path = tempfile.mkstemp(suffix=".csv")
         layer = IFACE.activeLayer()
         # select the first 2 features (the same used to produce the reference
         # csv)
         layer.select([1, 2])
         # probably we have the wrong layer selected (uhs produce many layers)
         self.viewer_dock.write_export_file(exported_file_path)
-        expected_file_path = os.path.join(
-            self.data_dir_name, 'hazard', expected_file_name)
-        self.assertTrue(
-            filecmp.cmp(exported_file_path, expected_file_path),
-            'The exported file (%s) is different with respect to the'
-            ' reference one (%s)' % (exported_file_path, expected_file_path))
+        with open(exported_file_path, 'r') as got:
```

Comment:
You could add a comment about why we are doing such relaxed check, for future memory.

Reply:
Done